source/dem/dem.cc

```diff
@@ -816,6 +810,80 @@ DEMSolver<dim>::post_process_results()
     }
 }
 
+template <int dim>
+void
+DEMSolver<dim>::report_statistics()
+{
+  // Update statistics on contact list
+  double number_of_list_built_since_last_log =
+    double(contact_build_number) - contact_list.total;
+  contact_list.max =
+    std::max(number_of_list_built_since_last_log, contact_list.max);
+  contact_list.min =
+    std::min(number_of_list_built_since_last_log, contact_list.min);
+  contact_list.total += number_of_list_built_since_last_log;
+  contact_list.average = contact_list.total /
+                         (simulation_control->get_step_number()) *
+                         simulation_control->get_log_frequency();
+
+  // Update statistics on time
```

Comment:
What does this comment mean here? There is no code that updates in time afterwards.

Reply:
100%  right. Fixed it :)!